schema/causality_jwt.js

```diff
@@ -47,7 +47,7 @@ export default {
       return Promise.all([
         gravity(`sale/${options.sale_id}`),
         gravity.with(accessToken)('me'),
-        gravity.with(accessToken)('me/bidders'),
+        gravity.with(accessToken)('me/bidders', { sale_id: options.sale_id }),
```

Comment:
Even better than my idea!

Although, doesn't the check below become simpler now?